rct_optimizations/CMakeLists.txt

```diff
@@ -20,10 +20,11 @@ add_library(${PROJECT_NAME} SHARED
   src/${PROJECT_NAME}/camera_intrinsic.cpp
   src/${PROJECT_NAME}/pnp.cpp
   src/${PROJECT_NAME}/multi_camera_pnp.cpp
-  # DH Chain
+  # DH Chain Kinematic Calibration
   src/${PROJECT_NAME}/dh_chain.cpp
+  src/${PROJECT_NAME}/dh_chain_kinematic_calibration.cpp
 )
-target_compile_options(${PROJECT_NAME} PUBLIC -std=c++14)
+target_compile_options(${PROJECT_NAME} PUBLIC -std=c++11)
```

Comment:
Previously I was using `std::make_unique` which is a C++14 feature; I removed the use of unique pointers in this PR, so I just reverted back to C++11 since 14 wasn't strictly necessary